functions/zoom-webhook.py

```diff
@@ -55,10 +56,13 @@ def handler(event, context):
             "Handling not implement for status '{}'".format(payload['status'])
         )
 
+    now = datetime.strftime(datetime.today(), '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
can this be in UTC?

Reply:
Sure. I didn't think about it. I was mainly including this to know how old deadletter messages so I was thinking more in terms of days or weeks rather then hours.